app/src/main/java/eu/faircode/email/Core.java

```diff
@@ -122,6 +123,8 @@
     private static final long YIELD_DURATION = 200L; // milliseconds
     private static final long MIN_HIDE = 60 * 1000L; // milliseconds
 
+    private static final FlagTerm FLAGGED = new FlagTerm(new Flags(Flags.Flag.FLAGGED), true);
```

Comment:
I don't think this is going to make a difference

Reply:
Yeah, probably not noticeable.